crates/rust-analyzer/src/world.rs

```diff
@@ -158,21 +175,7 @@ impl WorldState {
             });
         change.set_crate_graph(crate_graph);
 
-        // FIXME: Figure out the multi-workspace situation
```

Comment:
This fixme has not actually been adressed, would you mind moving it into `create_watcher`?

Reply:
Sure thing, moved.